easybuild/easyconfigs/l/LLVM/LLVM-3.6.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock='CMakeMake'
+
+name = "LLVM"
+version = "3.6.0"
+
+homepage = "http://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+# Do not set optarch to True: it will cause the build to fail
```

Comment:
any more details? what kind of issue would you run into (and any idea why exactly)?